styles.css

```diff
@@ -0,0 +1,103 @@
+body {
+  background-image: linear-gradient(115deg, rgba(58, 58, 158, 0.8), rgba(136, 136, 206, 0.7)), url(https://cdn.freecodecamp.org/testable-projects-fcc/images/survey-form-background.jpeg);
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: center;
+  text-align: center;
+}
+
+#title {
+  color: white;
+  font-family: 'Times New Roman', Times, serif;
+  font-size: 42px;
+  font-weight: 700;
+}
+
+#description {
+  color: white;
+  font-style: italic;
+  font-size: 20px;
+}
+
+main {
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  flex-direction: row;
+  align-items: center;
+}
+
+#container {
+  background-color: rgb(7, 7, 72, 50%);
+  width: 50%;
+  text-align: left;
+  padding: 50px;
+  font-size: 25px;
+  color: white;
+  border-radius: 8px;
+  margin: auto;
+}
+
+.sec-form ul {
+  list-style: none;
+}
+
+.sec-form li {
+  display: flex;
+  flex-direction: column;
+}
+
+.sec-form input {
+  width: 90%;
+  border-radius: 5px;
+  height: 30px;
+  margin-bottom: 20px;
+  font-size: 15px;
+}
+
+.sec-form #dropdown {
+  width: 90%;
+  border-radius: 5px;
+  height: 30px;
+  margin-bottom: 20px;
+  font-size: 15px;
+}
+
+.sec2-form ul {
+  list-style: none;
+}
+
+.radio_id {
+  display: flex;
+  flex-direction: row;
+}
+
+.sec3-form {
+  display: flex;
+  flex-direction: column;
+}
+
+.sec3-form label {
+  margin-left: 45px;
+}
+
+.sec3-form textarea {
+  width: 90%;
+  font-size: 15px;
+  height: 100px;
+  margin-top: 10px;
+  margin-bottom: 30px;
+  border-radius: 8px;
+  margin-left: 45px;
+}
+
+.sec-form button {
+  width: 90%;
+  background-color: green;
+  height: 40px;
+  border: 0;
+  color: white;
+  font-size: 20px;
+  border-radius: 8px;
+  margin-left: 45px;
+}
```

Comment:
- [ ] Great work using the inline comments. Based on the requirement for this exercise, you need to write at least two comments in this review. Kindly consider making more inline comments. 👍 😎 



#### Hint: I would advise you to highlight some other blocks of codes that need to be changed and make more inline comments

Reply:
- [ ] When making reviews, kindly consider the following factors:

### 1. Your comments should sound polite. 

#### Example of impolite comments:

> "Use semantic tags"

#### Example of polite comments:

> "I think it would be better to use semantic tags"

### 2. You should give a good reason why you are making the suggestion

#### Example of unexplained comment:

> Why don't you use an article tag here instead of a div?

#### Example of properly explained comment:

> Why don't you use an article tag for the .first-article element instead of a div, that would improve the HTML structure and make the page more accessible

### 3. Your comment should be clear and easy to understand

#### Example of vague comment: 

> The README is not descriptive, why don't you make it better?

#### Example of clear comment: The README's "Built With" and "Getting Started" sections are currently empty. Why don't you add content, listing the technology used in the project and how to run it, to them so your README is more informative and professional?

#### Hint: I would advise you to make your comments meet these criteria above. 👍 💻